### DISCUSSION TITLE
is there anyway to use with older swift toolchain

### DISCUSSION THREAD

**Discussion by robaho** [SwiftyLab/MetaCodable#109] [General]
My xcode only has 5.7 and the package requires 5.9. I manually downloaded the 5.9 toolchain and installed and selected it, but XCode still complains about the package.

**Comment by soumyamahunt** [comment]
Hi @robaho, this package relies on macro feature provided by Swift. Since the macro features were introduced with Swift 5.9, this package can only work with Swift 5.9 and above.
